hnn_core/viz.py

```diff
@@ -1040,3 +1040,61 @@ def _onclick(event):
 
     plt_show(show)
     return ax.get_figure()
+
+
+def plot_csd(csd,
+             times,
+             method,
+             filtered,
+             show_cb=True,
+             cmap="brw",
+             ax=None,
+             show=True):
+    """Plot synaptic weight of connections.
```

Comment:
You mean "plot CSD" ?

Reply:
Yes it's an copy-paste error of the doc-string. Will fix this.